make-options.json

```diff
@@ -12,6 +12,7 @@
         "AzureFileCopyV1",
         "AzureFileCopyV2",
         "AzureFunctionV1",
+        "AzureIoTEdge",
```

Comment:
rename the task folder to AzureIoTEdgeV2

Reply:
fixed